ocs_ci/ocs/acm/acm.py

```diff
@@ -4,19 +4,20 @@
 from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
@AviadP Please add your inputs

Reply:
i dont mind changing it according to Martin's suggestion, but in my opinion it would be better to complete this PR first and dedicate new one for location change

Reply:
+1

Reply:
>  but in my opinion it would be better to complete this PR first and dedicate new one for location change

agreed

Reply:
opened [issue](https://github.com/red-hat-storage/ocs-ci/issues/5360) for that

Reply:
@mbukatov @am-agrawa please mark the conversation as resolved